docs/pages/add.page.server.mdx

```diff
@@ -13,6 +13,8 @@ To add Vike to an existing Vite app:
 
    - <RepoLink text="Example" path="/boilerplates/boilerplate-vue/server/index.js" />
 
+{ /* NOTE(aurelien) modify the boilerplates to use the V1 design */ }
```

Comment:
@brillout I think I should do this in this same PR, so that the V1 docs and the V1 boilerplates land together. What do you think?

Reply:
Yes makes sense 👍

Reply:
done :) feel free to review `/boilerplates/**` in this PR if you feel like it

Reply:
Looks good 💯